pkg/apis/messaging/v1beta1/kafka_channel_types_test.go

```diff
@@ -33,7 +35,7 @@ func TestKafkaChannel_GetGroupVersionKind(t *testing.T) {
 	}
 }
 
-func TestKafkaChannelGetStatus(t *testing.T) {
+func TestKafkaChannel_GetStatus(t *testing.T) {
```

Comment:
nit: let's not the `_` - let's do camelCase

Reply:
The above instance was IntelliJ - not sure how this crept in though - good catch - thanks - will fix!